src/Nethermind/Nethermind.HealthChecks/HealthChecksPlugin.cs

```diff
@@ -40,6 +40,8 @@ public class HealthChecksPlugin : INethermindPlugin, INethermindServicesPlugin
 
         private ClHealthLogger _clHealthLogger;
 
+        private const int ClUnavailableReportMessageDelay = 5;
```

Comment:
hmmm shouldn't we copy it from HealthChecks config or shouldn't we have shared const ?

Reply:
this is time between warnings, if cl is not alive it will print warning every 5 seconds

Reply:
*it will check again every 5 seconds

Reply:
ok, I see :) LGTM